app/services/participant_validation_service.rb

```diff
@@ -19,15 +19,26 @@ def validate
     validated_record = matching_record(trn: trn, nino: nino, full_name: full_name, dob: date_of_birth)
     return if validated_record.nil?
 
-    {
+    validation_data = {
       trn: validated_record[:teacher_reference_number],
       qts: validated_record[:qts_date].present? && validated_record[:qts_date] != "null",
       active_alert: validated_record[:active_alert],
+      previous_participation: false,
+      previous_induction: false,
```

Comment:
What are these for? Are they always false?

Reply:
As we set them now, it's just initialising them so the setting logic is simpler

Reply:
Ah sorry, had misread below as setting `:ineligibility_flag`, not `ineligibility_flag`